commit 7156ac005bdcae789c0033d0c6e3b57f1f764a1c
Author: Martin Pitt <martin.pitt@ubuntu.com>
Date:   Tue Nov 29 19:06:49 2011 +0100

    75-persistent-net-generator.rules: Add Xen
    
    Xen devices are not always caught by the SUBSYSTEMS=="xen" clause, so explictly
    add their registered MAC prefix:
    
      http://www.coffer.com/mac_find/?string=00%3A16%3A3E
    
Bug-Ubuntu: https://launchpad.net/bugs/897789

diff --git a/extras/rule_generator/75-persistent-net-generator.rules b/extras/rule_generator/75-persistent-net-generator.rules
index 539807c..aab579c 100644
--- a/extras/rule_generator/75-persistent-net-generator.rules
+++ b/extras/rule_generator/75-persistent-net-generator.rules
@@ -61,6 +61,8 @@ ENV{MATCHADDR}=="52:54:4c:*", GOTO="globally_administered_whitelist"
 ENV{MATCHADDR}=="52:54:ab:*", GOTO="globally_administered_whitelist"
 # Kingston Technologies
 ENV{MATCHADDR}=="e2:0c:0f:*", GOTO="globally_administered_whitelist"
+# Xensource
+ENV{MATCHADDR}=="00:16:3e:*", GOTO="globally_administered_whitelist"
 
 # match interface dev_id
 ATTR{dev_id}=="?*", ENV{MATCHDEVID}="$attr{dev_id}"
